test/forkMainnet/LimitOrderSwap/CoordinatedTaker.t.sol

```diff
@@ -291,7 +295,7 @@ contract CoordinatedTakerTest is LimitOrderSwapTest {
             takerTokenAmount: defaultCrdOrder.takerTokenAmount,
             makerTokenAmount: defaultCrdOrder.makerTokenAmount,
             extraAction: bytes(""),
-            userTokenPermit: defaultPermit,
+            userTokenPermit: allowanceTransferPermit, // should be able to transfer without permit data
```

Comment:
It means no `AllowanceTransfer` data is needed since no need to call `Permit2.permit()` twice. But the whole permit struct is still needed.

Reply:
Got it. Would this work? This would make it more clear that no permit data is needed
```suggestion
            userTokenPermit: bytes(""), // should be able to transfer without permit data
```

Reply:
What about:
```suggestion
            userTokenPermit: allowanceTransferPermit, // should transfer from allowance directly
```

Reply:
Fixed in abfe3e1.